website/src/controllers/user.js

```diff
@@ -117,12 +117,26 @@ api.score = function(req, res, next) {
     }, saved.toJSON().stats));
 
     // Webhooks
+
+    // Select character data to send
+    var userData = _.pick(user.toJSON(), ['_id', '_tmp', 'stats']); // user.toJSON to copy-by-value
```

Comment:
I deleted my last comment, I didn't see you had posted in between.

That seems like a good-enough solution